src/bb-update.php

```diff
@@ -18,6 +18,10 @@ class BBPatch_24 extends BBPatchAbstract
 {
     public function patch()
     {
+        // Initializing meta description support for announcements
+        $q = "ALTER TABLE `post`  ADD `description` TEXT CHARACTER SET utf8 COLLATE utf8_general_ci NULL  AFTER `title`;"
```

Comment:
Bad syntax, semi colon isn't at the end of the line

Reply:
Fixed, thanks.